src/MLJInterface.jl

```diff
@@ -183,17 +183,35 @@ function update(mlj_model::MLJInterface.MODELS, verbosity::Int, fitresult, cache
     old_lgbm_model, old_classes, old_mlj_model = fitresult
 
     # we can continue boosting if and only if num_iterations has changed
-    if MLJModelInterface.is_same_except(old_mlj_model, mlj_model, :num_iterations)
-        additional_iterations = mlj_model.num_iterations - old_mlj_model.num_iterations
-        # eh this is ugly, possibly prompts a need for some refactoring
-        report = LightGBM.train!(old_lgbm_model, additional_iterations, String[], verbosity, LightGBM.Dates.now())
-        # should probably dump report too, its just an empty dict...
-        fitresult = (old_lgbm_model, old_classes, mlj_model)
-        return (fitresult, cache, (report,))
+    if !MLJModelInterface.is_same_except(old_mlj_model, mlj_model, :num_iterations)
+        # if we get here it's just means we need to call fit directly
+        return MLJInterface.fit(mlj_model, verbosity, X, y, w)
     end
 
-    # if we get here it's just means we need to call fit directly
-    return MLJInterface.fit(mlj_model, verbosity, X, y, w)
+    additional_iterations = mlj_model.num_iterations - old_mlj_model.num_iterations
+    if verbosity >= 1
+        @info("Not refitting from scratch", additional_iterations)
+    end
```

Comment:
Something missing in the log here? Like "Not refitting from scratch. Adding $additional_iterations iterations. "

Reply:
Log lines line this will actually do something of the form:
```
julia> @info "hello" buffer
┌ Info: hello
└   buffer = "buffer"

julia> 

```

Reply:
Great to know, thanks!